### DISCUSSION TITLE
Kanban (columns) view for Milestone

### DISCUSSION THREAD

**Discussion by brylie** [community/community#5552] [Projects and Issues]
Milestones currently display a list of tasks and allow sorting by various criteria. However, it would be useful to view progress in a milestone in a two-dimensional kanban "task board". E.g. showing columns of tasks based on their status: todo, in-progress, done.

I like how projects offer kanban functionality. But projects require additional steps to move tasks around. Whereas, Milestones are more closely situated with tasks.

**Comment by JoshHrach** [comment]
I would like this.

In my individual repos, I have a project set up for managing app development. In each project, I've been using a Single Select field to define a Version list. I can then have a board view of my versions and plan out what items and features will be going into future updates. 

As I'm working on several apps right now, I decided to make a single project where I can house all of my issues/work across my app repositories. However, I can't use the Single Field to define a version as each app has its own version. Instead, I was thinking I could start using Milestones in this regard. ("App One v1.1", "App Two v2.5", etc)

I'd still like to have a way to view these in a board, however, so that I can move issues easily between milestones as I plan out what's to be done for future releases.

**Comment by albjeremias** [comment]
bump. this would be an awesome feature. i wonder if there is an app or something for this. maybe some sync with nextcloud Deck app (kanban app)

**Comment by JoshHrach** [comment]
This is probably my number one feature request for Projects right now. I hope we'll see movement on this in 2026.
